src/extensions/extension-compatibility.ts

```diff
@@ -23,8 +23,12 @@ import semver from "semver";
 import { appSemVer } from "../common/vars";
 import type { LensExtensionManifest } from "./lens-extension";
 
-export function isCompatibleExtension(manifest: LensExtensionManifest): boolean {
+export function isCompatibleExtension(manifest: LensExtensionManifest, isBundled: boolean): boolean {
   if (manifest.engines?.lens) {
+    if (isBundled) {
+      return true;
+    }
+
```

Comment:
Comment that, since Lens v5, bundled extensions must specify the lens engine?
This assumes that we aren't going to generate any v4 versions from this code :)

Reply:
That is correct, we only ever generate a previous patch version from the tag (plus additonal commits) of the most recent patch version.